src/main/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidator.java

```diff
@@ -154,6 +155,20 @@ public void validateNationalitySection(HttpServletRequest request, List<ApiError
         validateNationalityLength(request, errorList, dto);
     }
 
+    public void validateOccupationSection(HttpServletRequest request, List<ApiError> errorList, OfficerFilingDto dto, AppointmentFullRecordAPI appointmentFullRecordAPI) {
+        if (Boolean.FALSE.equals(dto.getOccupationHasBeenUpdated())) {
+                return;
+        }
+        if (dto.getOccupation() == null) {
```

Comment:
Does a null occupation not mean we are removing the occupation? So we still need to check against CHIPS?

Reply:
or have we set it to NONE already?

Reply:
Null and None against chips data is checked doesOccupationMatchChipsData()

Reply:
But it's the same a NONE so should not be a change? We need to still cann doesOccupationMatchChips data and make that treat "",null and "NONE" all as the same.